examples/large_models/inferentia2/llama2/requirements.txt

```diff
@@ -0,0 +1,6 @@
+--extra-index-url https://pip.repos.neuron.amazonaws.com
+torch-neuronx==1.13.1.1.9.0
+transformers-neuronx==0.5.58
+transformers==4.31.0
```

Comment:
Instead of pinning the versions can these be changed to >= to indicate version x and higher

Reply:
As new releases of these packages come out, they may not work with other packages in this list and break the example, so I've pinned these packages to specific versions. Please let me know your thoughts.

Reply:
In that case should the installs in the README be pinned as well?

Reply:
Updated the Readme to install necessary packages using the included `requirements.txt` file.